apps/src/blockly/addons/contextMenu.js

```diff
@@ -249,6 +248,38 @@ const registerHighContrastTheme = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(highContrastThemeOption);
 };
 
+/**
+ * Change workspace theme to CdoAccessibleTheme
+ */
+const registerAccessibleTheme = function() {
+  const accessibleThemeOption = {
+    displayText: function(scope) {
+      return (
+        (isCurrentTheme(Themes.ACCESSIBLE, scope.workspace)
+          ? '✓ '
```

Comment:
I see this checkmark string a couple more times in this file - create a constant for it?

Reply:
In the follow-task linked in the overview, you'll see this is going to be refactored so it will only appear once!

Reply:
> @fisher-alice Currently, we're getting away with string literals for options like `registerDeletable` because those options are only available to levelbuilders via start mode. This could certainly be explained with a comment however, which I'm happy to add.

Thanks for the explanation! A comment would be helpful.